.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-*   @dotNomad @kgartland-rstudio @mmarchetti @sagerb
+*   @dotNomad @kgartland-rstudio @sagerb @marcosnav
```

Comment:
@mmarchetti 😢

Reply:
Should we pull this out into a separate PR, just to get the `CODEOWNERS` right as quickly as possible?

Reply:
I created https://github.com/posit-dev/publisher/pull/2278 to get this in so @mmarchetti doesn't get as many non-targeted Publisher notifications.

Reply:
Thank you!